src/features/broker-protection/comparisons/matches-full-address.js

```diff
@@ -46,8 +46,8 @@ export function matchesFullAddress (userAddresses, comparisonAddressFull, missin
         }
 
         if (!missingState &&
-          comparisonAddressFull.includes(userAddress.city) &&
-          comparisonAddressFull.includes(userAddress.state)
+          comparisonAddressFull.includes(userAddress.city.toLowerCase()) &&
+          comparisonAddressFull.includes(userAddress.state.toLowerCase())
         ) {
             return true
         }
```

Comment:
the address comparison, like others needs work. But this is a bug-fix to make this work how it was originally intended

Reply:
omg